tests/tasks/lib/test_sst.py

```diff
@@ -0,0 +1,149 @@
+from jiant.tasks import create_task_from_config_path
+from jiant.utils.testing.tokenizer import SimpleSpaceTokenizer
+
+
+TRAIN_EXAMPLES = [
+    {"guid": "train-0", "text": "hide new secretions from the parental units ", "label": "0"},
+    {"guid": "train-1", "text": "contains no wit , only labored gags ", "label": "0"},
+    {
+        "guid": "train-2",
+        "text": "that loves its characters and communicates something rather beautiful about human nature ",
+        "label": "1",
+    },
+    {
+        "guid": "train-3",
+        "text": "remains utterly satisfied to remain the same throughout ",
+        "label": "0",
+    },
+    {
+        "guid": "train-4",
+        "text": "on the worst revenge-of-the-nerds clich\u00e9s the filmmakers could dredge up ",
+        "label": "0",
+    },
+]
+
+TOKENIZED_TRAIN_EXAMPLES = [
+    {
+        "guid": "train-0",
+        "text": ["hide", "new", "secretions", "from", "the", "parental", "units"],
+        "label_id": 0,
+    },
+    {
+        "guid": "train-1",
+        "text": ["contains", "no", "wit", ",", "only", "labored", "gags"],
+        "label_id": 0,
+    },
+    {
+        "guid": "train-2",
+        "text": [
+            "that",
+            "loves",
+            "its",
+            "characters",
+            "and",
+            "communicates",
+            "something",
+            "rather",
+            "beautiful",
+            "about",
+            "human",
+            "nature",
+        ],
+        "label_id": 1,
+    },
+    {
+        "guid": "train-3",
+        "text": ["remains", "utterly", "satisfied", "to", "remain", "the", "same", "throughout"],
+        "label_id": 0,
+    },
+    {
+        "guid": "train-4",
+        "text": [
+            "on",
+            "the",
+            "worst",
+            "revenge-of-the-nerds",
+            "clich\u00e9s",
+            "the",
+            "filmmakers",
+            "could",
+            "dredge",
+            "up",
+        ],
+        "label_id": 0,
+    },
+]
+
+
+FEATURIZED_TRAIN_EXAMPLE_0 = {
+    "guid": "train-0",
+    "input_ids": np.array([1, 4, 5, 6, 7, 8, 9, 10, 2, 0]),
+    "input_mask": np.array([1, 1, 1, 1, 1, 1, 1, 1, 1, 0]),
+    "segment_ids": np.array([0, 0, 0, 0, 0, 0, 0, 0, 0, 0]),
+    "label_id": 0,
+    "tokens": ["<cls>", "hide", "new", "secretions", "from", "the", "parental", "units", "<sep>"],
+}
+
+
+def test_sst_create_examples():
+    task = create_task_from_config_path(
+        os.path.join(os.path.dirname(__file__), "resources/sst.json"), verbose=False
+    )
+    examples = task.get_train_examples()
+    for example_dataclass, raw_example_dict in zip(examples, TRAIN_EXAMPLES):
+        assert example_dataclass.to_dict() == raw_example_dict
+
+
+def test_sst_create_tokenized_examples():
+    task = create_task_from_config_path(
+        os.path.join(os.path.dirname(__file__), "resources/sst.json"), verbose=False
+    )
+    examples = task.get_train_examples()
```

Comment:
@pyeres the setup for this test and `test_sst_create_examples()` is the same. For my own learning, is it best to explicitly test a part of the code in each test or could 
'for example_dataclass, raw_example_dict in zip(examples, TRAIN_EXAMPLES):
        assert example_dataclass.to_dict() == raw_example_dict'
be included here and `test_sst_create_examples()` be removed?

Reply:
We discussed this a bit out-of-band. In general we'd like to shoot for the tests to have only one reason to fail, but we also want to make it as straightforward as possible (ccaa554 shows that the test implementations are less intuitive if we to try to force more separation between cases). So, we landed on a one test for task data loading through featurization (021944d).